infra/growth/src/resources/email.js

```diff
@@ -26,37 +26,67 @@ const reminderTextTemplate = fs
 const reminderHtmlTemplate = fs
   .readFileSync(`${templateDir}/emailReminder.html`)
   .toString()
+const payoutTextTemplate = fs
+  .readFileSync(`${templateDir}/emailPayout.txt`)
+  .toString()
+const payoutHtmlTemplate = fs
+  .readFileSync(`${templateDir}/emailPayout.html`)
+  .toString()
 
 /**
  * Returns the content for invite email.
  * TODO: localize the content.
  *
  * @param {string} emailType: 'invite' or 'reminder'
- * @param {string} friendName
+ * @param {Object} vars: dynamic variables
  * @param {string} targetUrl
  * @returns {{subject: string, html: *, text: *}}
  */
-function generateEmail(emailType, referrerName, targetUrl) {
-  let subject, textTemplate, htmlTemplate
-  if (emailType === 'invite') {
-    subject = 'Join Origin and earn free cryptocurrency'
-    textTemplate = inviteTextTemplate
-    htmlTemplate = inviteHtmlTemplate
-  } else {
-    subject = 'Earn Origin cryptocurrency'
-    textTemplate = reminderTextTemplate
-    htmlTemplate = reminderHtmlTemplate
+function generateEmail(emailType, vars) {
```

Comment:
I don't want to make it super fancy and generic because in future we should use the notification service rather than sending emails directly. But what you have is better. Will update...

Reply:
Yeah I don't feel strongly about my suggestion. I think both approaches are fine... I am ok with whatever you decide to do

Reply:
Added a TODO. This PR is getting huge ! Will tackle in a separate PR... :)